### PR TITLE
clean up logs, fix email notification listings

### PR DESCRIPTION
**Pull request by duttonw**

### CODE REVIEW

File: .docker/scripts/deactivate [hunk 1]
```diff
@@ -0,0 +1,3 @@
+if [ "$VENV_DIR" != "" ]; then
+  deactivate
```
Comment (RossWebsterWork) [inline .docker/scripts/deactivate:2]: Should this be `. ${VENV_DIR}/bin/deactivate` ?

**Review by RossWebsterWork — APPROVED** [review]

File: .docker/scripts/deactivate (hunk 1 above)
Comment (ThrawnCA) [reply .docker/scripts/deactivate:2]: Not necessary. If a virtualenv has been activated, then its bin directory is on the PATH. We shouldn't make assumptions about where that is, if we don't need to.

File: .docker/scripts/serve.sh [hunk 2]
```diff
@@ -5,11 +5,15 @@ dockerize -wait tcp://postgres:5432 -timeout 1m
 dockerize -wait tcp://solr:8983 -timeout 1m
 dockerize -wait tcp://redis:6379 -timeout 1m
 
-sed -i "s@SITE_URL@${SITE_URL}@g" $CKAN_INI
+for i in {1..60}; do
```
Comment (ThrawnCA) [inline .docker/scripts/serve.sh:8]: This syntax is actually not valid (my bad), see ckanext-qgov for a correct way of doing it.

**Review by ThrawnCA — CHANGES_REQUESTED** [review]

File: .ahoy.yml [hunk 3]
```diff
@@ -112,21 +112,22 @@ commands:
...
-      ahoy cli 'pytest --ckan-ini=${CKAN_INI}' || \
+      ahoy cli 'pytest --ckan-ini=${CKAN_INI} $APP_DIR/ckanext' || \
       [ "${ALLOW_UNIT_FAIL:-0}" -eq 1 ]
 
   test-bdd:
     usage: Run BDD tests.
     cmd: |
       ahoy start-mailmock &
-      if (echo "$CKAN_VERSION" | grep '^ckan-2.8'); then
+      if [ "$CKAN_VERSION" = "2.8" ]; then
```
Comment (ThrawnCA) [inline .ahoy.yml:130]: This is obsolete.

**Review by ThrawnCA — APPROVED** [review]

**Pull request merged**